commit dcda1c1721cfb7514faa521483a3a7bf069ad1ba
Author: Rob Johnston <rob.johnston@joyent.com>
Date:   2019-04-08T18:44:23+00:00 (6 months ago)
    
    OS-7309 build tools should support loader-based boot images (fix clean target)

diff --git a/tools/format_image/Makefile b/tools/format_image/Makefile
index 9ba19826..e8751be7 100644
--- a/tools/format_image/Makefile
+++ b/tools/format_image/Makefile
@@ -37,4 +37,4 @@ format_image: $(OBJS)
 	$(CC) -c $(CFLAGS) $(CPPFLAGS) -o $@ $<
 
 clean:
-	rm $(TARGETS) $(OBJS)
+	rm -f $(TARGETS) $(OBJS)
